frontend/pages/ManageControlsPage/MacOSSettings/cards/DiskEncryption/DiskEncryption.tsx

```diff
@@ -17,10 +19,18 @@ import DiskEncryptionTable from "./components/DiskEncryptionTable";
 const baseClass = "disk-encryption";
 interface IDiskEncryptionProps {
   currentTeamId?: number;
+  config: IConfig;
+  refetchProfiles: () => void;
+  refetchConfig: () => void;
 }
 
-const DiskEncryption = ({ currentTeamId }: IDiskEncryptionProps) => {
-  const { isPremiumTier, config } = useContext(AppContext);
+const DiskEncryption = ({
+  currentTeamId,
+  refetchProfiles,
+  refetchConfig,
+  config,
```

Comment:
Since we're modifying the config in these files, we need refetchConfig and useContext to be updated with a configAPI